spec/features/destroy_deck_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'deck/destroy' do
+  let!(:player) { Player.create(name: 'Karl') }
+  let!(:deck) { Deck.create(name: 'Llanowar Elf', player: player) }
```

Comment:
Prefer factory bot usage over straight active record to be future-proof / remove the need to update that spec if we add columns to the deck/player tables.

Reply:
This is correct - I changed it